code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -491,3 +491,131 @@
 	if(prob(15))
 		M.adjustToxLoss(2, 0)
 	..()
+
+/datum/reagent/drug/maint
+	name = "Maintanance Drugs"
+	addiction_type = /datum/reagent/drug/maint
+
+/datum/reagent/drug/maint/addiction_act_stage1(mob/living/M)
+	. = ..()
+	M.Jitter(1)
+
+/datum/reagent/drug/maint/addiction_act_stage2(mob/living/M)
+	. = ..()
+	M.Jitter(2)
+	if(prob(15))
+		M.emote(pick("twitch","drool"))
+
+/datum/reagent/drug/maint/addiction_act_stage3(mob/living/M)
+	. = ..()
+	M.Jitter(1)
+	M.adjustToxLoss(2)
+	if(prob(5))
+		M.drop_all_held_items()
+	if(prob(15))
+		M.emote(pick("twitch","drool"))
+
+/datum/reagent/drug/maint/addiction_act_stage4(mob/living/M)
+	. = ..()
+	M.Jitter(2)
+	M.adjustToxLoss(3)
+	if(prob(10))
+		M.drop_all_held_items()
+	if(prob(30))
+		M.emote(pick("twitch","drool"))
+
+/datum/reagent/drug/maint/powder
+	name = "Maintanance Powder"
+	description = "An unknown powder that you most likely gotten from an assistant, a bored chemist... or cooked yourself. It is a refined form of tar that enhances your mental ability, making you learn stuff a lot faster."
+	reagent_state = SOLID
+	color = "#ffffff"
+	metabolization_rate = 0.5 * REAGENTS_METABOLISM
+	overdose_threshold = 15
+	addiction_threshold = 5
```

Comment:
hm yes today I will addict random passerby to maintenance chems using a damp rag

Reply:
That's based

Reply:
this is actually an issue lol

Reply:
increased it to 6 u, no longer possible